### PR TITLE
Rb at pretty welcome

### LINKED ISSUE #14: 14. As a user, I want the welcome screen to look pretty so the app looks professional and welcoming.
AC:

* The welcome screen (the screen a user sees before they’ve created their list) looks something like the following (or equivalent, use your creativity!):

welcome-screen

### PR DESCRIPTION
**Pull request by audracodes**
## Description

Implemented styling library [Chakra UI ](https://chakra-ui.com/docs/getting-started) in the app and focused on styling the welcome screen. Created reusable styling to be used with the other components in the app.

## Related Issue

Closes #14 

## Acceptance Criteria

- [x] The welcome screen (the screen a user sees before they’ve created their list) looks something like the following (or equivalent, use your creativity!)

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  | :sparkles: New feature     |
| ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
| ✓ | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

**General Updates**
- Imported Chakra UI
- Imported Montserrat font through GoogleFonts
- Deleted `App.css`, `Welcome.css`
-Removed the wrapping `div` in `Router.js`

**App.js**
Wrapped the `BroswerRouter` in the `ChakraProvider` in order to use Chakra UI throughout the app. Imported our custom theme through the `ChakraProvider`. Updated the file to use Chakra UI styled components, and provide consistent styling for the header throughout the app.

**ViewList.js**
Changed 'View List' from an `h1` to an `h2` for more semantic html.

**Welcome.js**
Updated the file to use Chakra UI styled components and removed unnecessary comments. Error message is now displayed using the `Alert` component. 

**smartListTheme.js**
Created a custom theme for the app. Added brand colours, font weights, and began to create reusable `textStyles` and component styles.  

### Before

Screen Shot 2020-11-20 at 12 29 34 AM

### After

Screen Shot 2020-11-20 at 12 14 32 AM

Screen Shot 2020-11-20 at 12 14 46 AM

## Testing Steps / QA Criteria

App Welcome screen should display with styling matching the attached screenshots. An error message should appear when an incorrect token is submitted.

### CODE REVIEW

File: src/components/Welcome.js [hunk 1]
```diff
@@ -35,30 +51,53 @@ export default function Welcome() {
...
+              <VisuallyHidden>
+                <FormLabel htmlFor="tokenField"> Enter Token </FormLabel>
+              </VisuallyHidden>
+              <Input
+                variant="flushed"
+                w="30%"
+                textAlign="center"
+                textStyle={error ? 'error' : ''}
+                id="tokenField"
+                placeholder="three word token"
```
Comment (indiragp) [inline src/components/Welcome.js:79]: Maybe I am too OCD, but can the first word start with a capital ? "Three word token" 

Although I think I would just refer to it as a token, we shouldn't give too much details about what it is like :)

**Review by indiragp — APPROVED** [review]
Great Job! This looks really really pretty :) Great Color scheme as well!
If you are looking to use more than one color I recommend https://coolors.co/ generator, you can set some of the colors to the ones you match and it generates other ones. 
It may be helpful for probably the colors in the item list page or for the delete type of buttons :)

**Review by snsebro — APPROVED** [review]
This looks great! Thanks for adding the styles to our themes think we're in a great place to pick up with next week's issues.

**Comment by rachel-fischoff** [comment]
Hi! Everything looks good. Just curious what the @emotion dependencies are? We don't seem to use them in the app. :)

File: src/components/Welcome.js (hunk 1 above)
Comment (audracodes) [reply src/components/Welcome.js:79]: > Hi! Everything looks good. Just curious what the @emotion dependencies are? We don't seem to use them in the app. :)

Looking at the docs, the emotion dependencies seem to be part of the default install (https://chakra-ui.com/docs/getting-started)

**Comment by audracodes** [comment]
> Hi! Everything looks good. Just curious what the @emotion dependencies are? We don't seem to use them in the app. :)

Thank you! Looking at the docs, the emotion dependencies seem to be part of the default install https://chakra-ui.com/docs/getting-started

**Pull request merged**